docs/source/tutorials/regression.ipynb

```diff
@@ -0,0 +1,972 @@
+{
+ "cells": [
```

Comment:
Appreciate including the outputs of the notebook for easier review!  Don't forget to clear them before we merge this